### PR TITLE
ArgoCD toolset

### PR DESCRIPTION
**Pull request by nherment**

### CODE REVIEW

File: holmes/plugins/toolsets/argocd.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+toolsets:
+  - name: "argocd/core"
+    prerequisites:
+      - command: "argocd version"
+
+    tools:
+      - name: "argocd_app_get"
+        description: "Retrieve information about an existing application, such as its status and configuration"
+        command: "argocd app get {{ app_name }} --show-operation -o yaml --auth-token {{ auth_token }}"
```
Comment (arikalon1) [inline holmes/plugins/toolsets/argocd.yaml]: I think we don't want to auth token to be sent to the llm
Can we somehow pass the token via env var?
So, for example, in order to use the tool, the user need to inject the env var: ARGO_AUTH_TOKEN,
and the tool command will be:
```
command: "argocd app get {{ app_name }} --show-operation -o yaml --auth-token $ARGO_AUTH_TOKEN"
```
Can something like this work?

Comment (nherment) [reply holmes/plugins/toolsets/argocd.yaml]: Turns out we don't event need to use `--auth-token` as argocd will read the env var `ARGO_AUTH_TOKEN` by itself. Reflected in the documentation.

File: holmes/plugins/toolsets/argocd.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+toolsets:
+  argocd/core:
+    description: "Set of tools to get argocd metadata like list of apps, repositories, projects, etc."
+    docs_url: "https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd/"
+    icon_url: "https://argo-cd.readthedocs.io/en/stable/assets/logo.png"
+    tags:
+      - core
+    prerequisites:
```
Comment (itisallgood) [inline holmes/plugins/toolsets/argocd.yaml:8]: Should we add `ARGOCD_AUTH_TOKEN`env  as one of the prerequisites ?

**Review by itisallgood — APPROVED** [review]
lgtm

**Pull request merged**